.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 npm-debug.log
 lerna-debug.log
+yarn.lock
```

Comment:
This disappeared somewhere along the way. This is to ensure no one accidentally checks in a Yarn lock file if they're using Yarn instead of npm